### ISSUE TITLE
Changes in Liquid Room depth via Set_Liquid_Depth not maintained between loads

### ISSUE THREAD

**Issue by GooberRF** [rafalh/dashfaction#97]
To replicate:
-Open any map with a Set_Liquid_Depth event (like geothermal plant - L5S2)
-Trigger a Set_Liquid_Depth event (in L5S2, either lava control or water control will work)
-Save the game either while the Set_Liquid_Depth event is in progress, or after it's finished
-Load that save file
-The liquid depth will be the same as it was before triggering the event

I'm wondering if the liquid height parameter isn't saved in the .svl savegame file?

**Comment by rafalh** [comment]
> I'm wondering if the liquid height parameter isn't saved in the .svl savegame file?
That's correct. As far as I remember state of that event is not stored into a savegame file. Fixing it would require to change save format which is not something I'd like to do considering that this event is only used in one place in the game.
